src/appleseed/renderer/kernel/lighting/scatteringmode.h

```diff
@@ -70,7 +70,7 @@ class ScatteringMode
     static bool has_volume(const int modes);
     static bool has_diffuse_or_glossy_or_volume(const int modes);
     static bool has_diffuse_and_glossy(const int modes);
-    static bool has_diffuse_or_volumetric(const int modes);
+    static bool has_diffuse_or_Volume(const int modes);
```

Comment:
Should be `has_diffuse_or_volume()` with a lower case *v*.

That smells global search & replace :)